pkg/pillar/types/types.go

```diff
@@ -116,8 +116,14 @@ func (state SwState) ZSwState() info.ZSwState {
 		return info.ZSwState_RESOLVED_TAG
 	case DOWNLOADING:
 		return info.ZSwState_DOWNLOAD_STARTED
-	case DOWNLOADED, VERIFYING, VERIFIED, LOADING:
+	case DOWNLOADED:
 		return info.ZSwState_DOWNLOADED
+	case VERIFYING:
+		return info.ZSwState_VERIFYING
+	case VERIFIED:
+		return info.ZSwState_VERIFIED
+	case LOADING:
+		return info.ZSwState_LOADING
 	case LOADED:
 		return info.ZSwState_DELIVERED
```

Comment:
Shouldn't this use ZSwState_LOADED so we can retire DELIVERED?

Reply:
We need to do some cloud changes first before changing DELIVERED to LOADED.

Reply:
OK